### DISCUSSION TITLE
Is there add-on or plugin support for the Game Studio?

### DISCUSSION THREAD

**Discussion by martinrhan** [stride3d/stride#2065] [General]
I have to do a lot of operation that follows a certain pattern. It's nice if I can write a program to let it be done automatically.

**Comment by Nicogo1705** [comment]
With the avalonia rewrite, add-on will be a feature. ( https://github.com/stride3d/stride/pull/2034 )
For now, depends on what you want to achieve, you can make a processor and "mod" certains things. ( https://github.com/Doprez/lod-example )
